docs/products/postgresql/howto/setup-logical-replication.rst

```diff
@@ -175,4 +175,6 @@ For further information about WAL and checkpoints, read the `PostgreSQL document
 
 .. note::
 
-    The recreation of replication slots gets enabled automatically. Replication slots are recreated when a maintenance update is applied or a failover occurs, but they are not recovered after major version upgrades.
+    The recreation of replication slots gets enabled automatically for services created or updated as of January 2023.  Additional details are outlined on `our blog post <https://aiven.io/blog/aiven-for-pg-recreates-logical-replication-slots>`_. 
+    
+    Replication slots are recreated when a maintenance update is applied or a failover occurs, but they are not recovered after major version upgrades.
```

Comment:
`or a failover occurs` this applies only for multi-node setups, if one node service crashes in case of say cloud provider failure, the slots won't be recovered on a new node as this information is not stored anywhere except the dead node.

Reply:
edited to say `or a failover occurs (for multi-node clusters)` to make it more specific